docs/documentation/Window Functions.md

```diff
@@ -0,0 +1,237 @@
+# PartiQL Window Function Doc
+
+Window function:
+PartiQL currently supports SQL’s window functions with in-line window specification, i.e. `f(x) OVER( [window specification])`.  Note that frame clause is not currently supported ( but will be in the future).
```

Comment:
Maybe `This PartiQL implementation` or something like that?

Also `( but will be in the future)` -> `(but will be in the future)`